src/libs/actions/IOU.ts

```diff
@@ -364,6 +364,20 @@ function initMoneyRequest(reportID: string, policy: OnyxEntry<OnyxTypes.Policy>,
     });
 }
 
+/**
+ *
+ * @param transactionID
+ * @param reportID
+ */
+function resetMoneyRequest(transactionID: string, reportID: string, isFromGlobalCreate: boolean) {
```

Comment:
This function does not reset the money request, only the participants and the report id. Can you please find a better fitting name?